src/sentry/integrations/github_enterprise/integration.py

```diff
@@ -263,6 +280,212 @@ def on_create_or_update_comment_error(self, api_error: ApiError, metrics_base: s
 
         return False
 
+    def get_config_data(self):
+        config = self.org_integration.config
+        project_mappings = IntegrationExternalProject.objects.filter(
+            organization_integration_id=self.org_integration.id
+        )
+        sync_status_forward = {}
+
+        for pm in project_mappings:
+            sync_status_forward[pm.external_id] = {
+                "on_unresolve": pm.unresolved_status,
+                "on_resolve": pm.resolved_status,
+            }
+        config["sync_status_forward"] = sync_status_forward
+        return config
+
+    def _get_organization_config_default_values(self) -> list[dict[str, Any]]:
+        """
+        Return configuration options for the GitHub integration.
+        """
+        config: list[dict[str, Any]] = []
+
+        if features.has("organizations:integrations-github-project-management", self.organization):
```

Comment:
@iamrajjoshi this is a valid one. We want this to be keyed by GHE instead of GH right?